visualization/FairnessDashboard/src/IFairnessProps.ts

```diff
@@ -54,9 +54,9 @@ export interface IFairnessProps {
     theme?: any;
     locale?: string;
     stringParams?: any;
-    supportedBinaryClassificationAccuracyKeys?: string[];
-    supportedRegressionAccuracyKeys?: string[];
-    supportedProbabilityAccuracyKeys?: string[];
+    supportedBinaryClassificationAccuracyKeys: string[];
+    supportedRegressionAccuracyKeys: string[];
+    supportedProbabilityAccuracyKeys: string[];
```

Comment:
I think having them optional works best for the precomputed case. For the pyhton case, they are required.